plugins/modules/system/java_keystore.py

```diff
@@ -465,11 +513,10 @@ def create_file(content):
 def hex_decode(s):
     if PY2:
         return s.decode('hex')
-    else:
-        return s.hex()
+    return s.hex()
 
 
-class ArgumentSpec(object):
+class ArgumentSpec:
```

Comment:
I'd rather move all the `ArgumentSpec` class content into `main()`, but I thought in another PR :)

I removed the `(object)` because useless, reported by pylint, and for consistency (with JavaKeystore class above).

Reply:
Sure, either works for me.

Reply:
@quidame I think the `(object)` is only useless in Python 3, not in Python 2. Though for such simple code it shouldn't really make a difference...